scvi/model/base/_base_model.py

```diff
@@ -113,16 +175,70 @@ def get_anndata_manager(
                 )
             return None
 
-        adata_uuid = adata.uns[_SCVI_UUID_KEY]
-        if adata_uuid not in cls.manager_store:
+        adata_id = adata.uns[_SCVI_UUID_KEY]
+        manager_store_key = _ManagerStoreKey(adata_id=adata_id, model_instance_id=None)
```

Comment:
why is model instance id none here?

Reply:
oh wait I get it now... this is why you don't overwrite the key earlier?

Reply:
yes they have two different meanings. I tried to explain it here but agree its not straightforward https://github.com/YosefLab/scvi-tools/pull/1342/files#diff-41fbf1b79b096021b95dc02658ce6c30c499f58e43904cbb5a986b99ec02f747R41-R51